### ISSUE TITLE
Loading TOC / BIN files doesn't work ( FC rawhide - copr )

### ISSUE THREAD

**Issue by crvisqr** [cdemu/cdemu#3]
I've used cdemu a year back, and loading TOC / BIN audio cd image was working perfectly in my debian unstable. Now, I tried the same in FC rawhide in virtualbox ( installed from copr - https://copr.fedorainfracloud.org/coprs/rok/cdemu/ ) with the same TOC / BIN files. But, nothing is loaded in nautilus / rhythmbox. Nothing changes even after reboot.

```
[root@localhost ~]# ps -aef | grep cdem
crvi        2669    2493  0 16:52 pts/1    00:00:21 /usr/bin/python3 /usr/bin/gcdemu
crvi        2679    1579  0 16:52 ?        00:00:03 cdemu-daemon --ctl-device=/dev/vhba_ctl --bus=session --num-devices=1 --audio-driver=default --logfile=/home/crvi/.cdemu-daemon.log
root       12056    2994  0 22:43 pts/3    00:00:00 grep --color=auto cdem
```

```
[root@localhost ~]# cat /home/crvi/.cdemu-daemon.log
Starting CDEmu daemon with following parameters:
 - num devices: 1
 - control device: /dev/vhba_ctl
 - audio driver: default
 - bus type: session
 - default CDEmu debug mask: 0x0
 - default libMirage debug mask: 0x0

_g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
```

Thanks !!

**Comment by rokm** [comment]
This looks like a rawhide kernel problem to me. 

With ```5.6.0-0.rc0.git5.1.fc32.x86_64``` (which got installed as I updated my rawhide VM today), a loaded audio image does not appear in Nautilus or Ryhthmbox for me, either. But it works as expected with ```5.5.0-0.rc4.git2.1.fc32.x86_64``` (which was installed last time I used the VM).

I get the same behavior with my USB optical drive and a physical audio disc. 

Also with ```5.6.0-0.rc0.git5.1.fc32.x86_64```, packet commands seem to be failing with ```Bad address``` on VBox optical drive, CDEmu virtual device, and my USB optical drive.

For example:
```
[rok@vm-fedora-rawhide ~]$ cdrdao read-toc --device /dev/sr1 toc.toc
Cdrdao version 1.2.4 - (C) Andreas Mueller <andreas@daneb.de>
ERROR: SG_SET_RESERVED_SIZE ioctl failed: Bad address
ERROR: /dev/sr1: SCSI command Inquiry (0x12) failed: Bad address.
ERROR: Inquiry command failed on "/dev/sr1"
/dev/sr1: �&~4�U 	Rev: 
ERROR: Inquiry failed and no driver id is specified.
ERROR: Please use option --driver to specify a driver id.
ERROR: Cannot setup device /dev/sr1.
```

**Comment by crvisqr** [comment]
I've created https://bugzilla.kernel.org/show_bug.cgi?id=206491

**Comment by rokm** [comment]
Seems to work as expected on rawhide with ```5.6.0-0.rc1.git2.1.fc33.x86_64``` now.

**Comment by rokm** [comment]
Relevant entry from kernel package's changelog:
```
* Thu Feb 13 2020 Jeremy Cline <jcline@redhat.com>
- Pull in cdrom ioctl fix (rhbz 1801353)
```

**Comment by crvisqr** [comment]
Works. Thanks !
